### DISCUSSION TITLE
is support multi cte

### DISCUSSION THREAD

**Discussion by jiangpan** [maxtoroq/DbExtensions#64] [Q&A] [answered]
example: 
with a as( select col1 ,col2  from  table1 )
, b as ( select col3,col4 from table2 )
,c as (select col1,col2,col3,col4 from a inner join b on a.col1 = b.col4)
select * from c

**Comment by maxtoroq** [comment, marked as answer]
SqlBuilder only provides methods for basic clauses that most SQL dialects support. However, you can [extend SqlBuilder](https://maxtoroq.github.io/DbExtensions/docs/SqlBuilder.html#extensibility) using extension methods.
